src/components/application_manager/rpc_plugins/rc_rpc_plugin/src/resource_allocation_manager_impl.cc

```diff
@@ -223,7 +223,7 @@ EnumType StringToEnum(const std::string& str) {
   return val;
 }
 
-void ConstructOnRCStatusNotificationParams(
+void ResourceAllocationManagerImpl::ConstructOnRCStatusNotificationParams(
```

Comment:
@LitvinenkoIra do you need to make it member function just for call is_rc_enabled()? I would better make it additional param.